src/test/java/pl/byczazagroda/trackexpensesappbackend/service/FTCategoryUpdateServiceImplTest.java

```diff
@@ -30,9 +32,8 @@
 public class FTCategoryUpdateServiceImplTest {
 
     private static final long VALID_ID = 1L;
-
     private static final long INVALID_ID = 10L;
-
+    private static final long USER_ID_1L = 1L;
```

Comment:
Please insert a blank line between the class fields.

Reply:
Done.